mcs/tools/exception-filter-rewriter/Program.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using Mono.Cecil;
+using Mono.Cecil.Cil;
+
+namespace ExceptionRewriter {
+    class Program {
+        public static int Main (string[] _args) {
+            try {
+                var options = new RewriteOptions();
+
+                foreach (var arg in _args)
+                    ParseArgument(arg, options);
+
+                var argv = _args.Where(arg => !arg.StartsWith("-")).ToArray();
+
+                var step = options.Overwrite ? 1 : 2;
+                if (argv.Length < step) {
+                    Usage();
+                    return 1;
+                }
```

Comment:
Looks like most of this file isn't formatted to the [Mono Coding Guidelines](https://www.mono-project.com/community/contributing/coding-guidelines/) - should be tabs (width=8) not spaces; and braces for methods go on a new line; space before `(` of a method call; etc.